src/app/player/player.page.ts

```diff
@@ -48,57 +62,78 @@ export class PlayerPage implements OnInit, OnDestroy, PlayerActionHandlerDelegat
     (window as any).onUserSwitch = this.onUserSwitch.bind(this);
 
     if (this.router.getCurrentNavigation().extras.state) {
+      this.content = this.router.getCurrentNavigation().extras.state.contentToPlay;
       this.config = this.router.getCurrentNavigation().extras.state.config;
       this.course = this.router.getCurrentNavigation().extras.state.course;
       this.navigateBackToContentDetails = this.router.getCurrentNavigation().extras.state.navigateBackToContentDetails;
       this.corRelationList = this.router.getCurrentNavigation().extras.state.corRelation;
       this.isCourse = this.router.getCurrentNavigation().extras.state.isCourse;
+      this.isChildContent = this.router.getCurrentNavigation().extras.state.childContent;
     }
   }
 
   ngOnInit() {
-    this.pauseSubscription =  this.platform.pause.subscribe(() => {
+
+    if (this.config['metadata']['mimeType'] === 'application/pdf') {
```

Comment:
Why a separate configuration for pdf just attach the following in the existing config.
`sideMenu: {
            showShare: true,
            showDownload: true,
            showReplay: false,
            showExit: true,
          }`

Reply:
pdf player accepts config and meta data , and all other things are not required to pass to the new pdf player